web-portal/nuxt.config.js

```diff
@@ -2,6 +2,10 @@ import colors from 'vuetify/es5/util/colors'
 import 'dotenv/config'
 import { json } from 'body-parser'
 
+const API_URL = process.env.API_URL
+
+console.log('!!! ahh: ' + API_URL)
+
```

Comment:
Did you mean to leave this? If so, can we make the message clearer?

Reply:
whoops, haha I was about to say: "At least there are no swear words this time", then I found this one

```
console.log('!!! what the shit')
console.log(VenueModel)
console.log("_________")
```

Reply:
cleaning it up now :-)